apps/server/src/modules/tool/context-external-tool/uc/context-external-tool.uc.ts

```diff
@@ -97,6 +99,7 @@ export class ContextExternalToolUc {
 		userId: EntityId,
 		tools: ContextExternalTool[]
 	): Promise<ContextExternalTool[]> {
+		// authorizationService.getUserWithPermissions hould be part of the public method at place where ressources are loaded and pass the authorizable user to it.
```

Comment:
Yes, I could guess that it should say. What i didnt got is the "place where resources are loaded".

Reply:
I rewrite it to 2 sentence in the hope that the grammatic is better. :D 

"The UC should be the place for orchstration of loading, passing ressources. 
You can have a service that solve different types and use the currect source for loading. 
You can have services that combine stuff by loading in a specifc way an mapping. 
But the UC is the place that must have the over all controlling over the data flows. 
Otherwise we reduce the performance and do not see why.